tests/features/steps/helpers.py

```diff
@@ -73,6 +73,23 @@ def assert_is_now(val, key):
 def format_date(date_to_format):
     return date_to_format.strftime(DATETIME_FORMAT)
 
+def change_null_to_none(response_data):
+    """Checks for locking parameters in response_data and changes null values to string "None" for comparison
+    :response_data: dict of response_data
+    """
+    try:
```

Comment:
There are special values that can be used for this type of assertion. In this case `__none__` should work.
More can be found here: https://github.com/superdesk/sams/blob/develop/tests/features/steps/helpers.py#L125